From 43d81b3686d924e7413de19716d84e54054a42b1 Mon Sep 17 00:00:00 2001
From: August <2819763+Dangku@users.noreply.github.com>
Date: Sat, 13 Jul 2024 17:51:15 +0800
Subject: [PATCH] sunxi h616 rgmii pins

Signed-off-by: August <2819763+Dangku@users.noreply.github.com>
---
 arch/arm/dts/sun50i-h616.dtsi | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/dts/sun50i-h616.dtsi b/arch/arm/dts/sun50i-h616.dtsi
index 74aed0d232a..335c3fc93f5 100644
--- a/arch/arm/dts/sun50i-h616.dtsi
+++ b/arch/arm/dts/sun50i-h616.dtsi
@@ -162,8 +162,7 @@
 			ext_rgmii_pins: rgmii-pins {
 				pins = "PI0", "PI1", "PI2", "PI3", "PI4",
 				       "PI5", "PI7", "PI8", "PI9", "PI10",
-				       "PI11", "PI12", "PI13", "PI14", "PI15",
-				       "PI16";
+				       "PI11", "PI12", "PI13", "PI14", "PI15";
 				function = "emac0";
 				drive-strength = <40>;
 			};
-- 
2.25.1

